recipe/meta.yaml

```diff
@@ -1,25 +1,27 @@
 {% set name = "langdetect" %}
-{% set version = "1.0.9" %}
+{% set version = "1.0.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: cbc1fef89f8d062739774bd51eda3da3274006b3661d199c2655f6b3f6d605a0
+  # standard pypi url for 1.0.7 is missing
+  url: https://files.pythonhosted.org/packages/59/59/4bc44158a767a6d66de18c4136c8aa90491d56cc951c10b74dd1e13213c9/langdetect-1.0.7.zip
```

Comment:
It's available, just as a zip instead of a tarfile. https://pypi.io/packages/source/l/langdetect/langdetect-1.0.7.zip.